ports/librdkafka/portfile.cmake

```
@@ -17,34 +17,34 @@ vcpkg_check_features(OUT_FEATURE_OPTIONS FEATURE_OPTIONS
     snappy WITH_SNAPPY
 )
 
-vcpkg_configure_cmake(
+vcpkg_cmake_configure(
     SOURCE_PATH ${SOURCE_PATH}
-    PREFER_NINJA
     OPTIONS
         -DRDKAFKA_BUILD_STATIC=${RDKAFKA_BUILD_STATIC}
         -DRDKAFKA_BUILD_EXAMPLES=OFF
         -DRDKAFKA_BUILD_TESTS=OFF
         -DWITH_BUNDLED_SSL=OFF
         ${FEATURE_OPTIONS}
     OPTIONS_DEBUG
+        -DENABLE_SHAREDPTR_DEBUG=ON
         -DENABLE_DEVEL=ON
         -DENABLE_REFCNT_DEBUG=ON
         -DENABLE_SHAREDPTR_DEBUG=ON
         -DWITHOUT_OPTIMIZATION=ON
     OPTIONS_RELEASE
+        -DENABLE_SHAREDPTR_DEBUG=OFF
         -DENABLE_DEVEL=OFF
         -DENABLE_REFCNT_DEBUG=OFF
         -DENABLE_SHAREDPTR_DEBUG=OFF
         -DWITHOUT_OPTIMIZATION=OFF
 )
 
-vcpkg_install_cmake()
+vcpkg_cmake_install()
 
 vcpkg_copy_pdbs()
 
-vcpkg_fixup_cmake_targets(
+vcpkg_cmake_config_fixup(
     CONFIG_PATH lib/cmake/RdKafka

```

```suggestion
vcpkg_cmake_config_fixup(
    PACKAGE_NAME RdKafka
    CONFIG_PATH lib/cmake/RdKafka
```
We might also need to add `PACKAGE_NAME` to move config and targets cmake files to _share/RdKafka_.